configuration/types.go

```diff
@@ -60,6 +60,8 @@ const (
 	DefaultBlockBroadcastLimit               = 5
 	DefaultStatusPort                        = 9090
 	DefaultMaxReorgDepth                     = 100
+	DefaultMaxTableSize                      = 256 << 20
```

Comment:
Eventhough you left a comment below that explains what size this is, could you leave a note here too?